crates/task/src/lib.rs

```diff
@@ -271,3 +271,25 @@ pub struct TaskContext {
 /// This is a new type representing a 'tag' on a 'runnable symbol', typically a test of main() function, found via treesitter.
 #[derive(Clone, Debug)]
 pub struct RunnableTag(pub SharedString);
+
+/// Convert `${SOME_VAR}`, `$SOME_VAR` to `$env:SOME_VAR`.
+#[inline]
+#[cfg(target_os = "windows")]
+pub fn to_powershell_variable(input: String) -> String {
+    if let Some(var_str) = input.strip_prefix("${") {
+        if var_str.find(':').is_none() {
+            // If the input starts with "${", remove the trailing "}"
+            format!("$env:{}", &var_str[..var_str.len() - 1])
+        } else {
+            // `${SOME_VAR:-SOME_DEFAULT}`, we currently do not handle this situation,
+            // which will result in the task failing to run in such cases.
+            input
+        }
+    } else if let Some(var_str) = input.strip_prefix('$') {
+        // If the input starts with "$", directly append to "$env:"
+        format!("$env:{}", var_str)
+    } else {
+        // If no prefix is found, return the input as is
+        input
+    }
+}
```

Comment:
Here, we only handle `$SOME_VAR` and `${SOME_VAR}`, but not default parameters like `${SOME_VAR:-SOME_DEFAULT}`. Handling such cases in PowerShell is pretty complex. In lower versions of PowerShell, you would use if-else statements, while in higher versions, you can use a ternary-like syntax. See [here](https://stackoverflow.com/questions/61548561/set-powershell-variable-with-environment-variable-or-a-default-if-not-present).